### DISCUSSION TITLE
Updating error

### DISCUSSION THREAD

**Discussion by taeyoonlee87** [r-lidar/lidR#636] [Q&A] [answered]
hello,

I tried to update lidR package (from 3.1.2 to 4.0.1) but it seems like kept facing an error. 

```
installing to C:/Users/XXX/Documents/R/win-library/3.6/00LOCK-lidR/00new/lidR/libs/x64
** R
** inst
** byte-compile and prepare package for lazy loading
Error: object 'wkt' is not exported by 'namespace:raster'
Execution halted
ERROR: lazy loading failed for package 'lidR'
* removing 'C:/Users/XXX/Documents/R/win-library/3.6/lidR'
* restoring previous 'C:/Users/XXX/Documents/R/win-library/3.6/lidR'
Warning in install.packages :
  installation of package ‘lidR’ had non-zero exit status
```

I have no issue when I update it using my desktop but my laptop has this issue. 

Could you please give me any advice?

**Comment by Jean-Romain** [comment, marked as answer]
Reinstall raster, if if does not work reinstall terra, if it does not work reinstall R. This error is definitively irrelevant and looks like there is a corruption somewhere. We sometime get stuff like that on windows for unknow reasons

**Reply by taeyoonlee87** [reply]
After I reinstalled the R, it fixed. Thanks!
